plugins/woocommerce/src/Utilities/DBUtil.php

```diff
@@ -0,0 +1,46 @@
+<?php
+/**
+ * A class of utilities for dealing with Database management.
+ */
+
+namespace Automattic\WooCommerce\Utilities;
+
+use Automattic\Jetpack\Constants;
+
+class DBUtil {
+
+	/**
+	 * Verify if the table(s) already exists.
+	 *
+	 * @param string $table_creation_sql Schema definition to check against.
+	 *
+	 * return array List of missing tables.
+	 */
+	public static function verify_database_tables_exist( $table_creation_sql ) {
```

Comment:
What about naming this `get_missing_database_tables` instead? This way it's more clear what the returned value is.